packages/account/src/Components/unlink-account-modal/__tests__/unlink-account-modal.spec.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import UnlinkAccountModal from '../unlink-account-modal';
+import userEvent from '@testing-library/user-event';
+
+describe('UnlinkAccountModal', () => {
+    let modal_root_el: HTMLDivElement;
+
+    beforeAll(() => {
+        modal_root_el = document.createElement('div');
+        modal_root_el.setAttribute('id', 'modal_root');
+        document.body.appendChild(modal_root_el);
+    });
+
+    afterAll(() => {
+        document.body.removeChild(modal_root_el);
+    });
+
+    const mock_props = {
+        onClose: jest.fn(),
+        is_open: true,
+        identifier_title: 'Google',
+        onClickSendEmail: jest.fn(),
+    };
+
+    it('should render modal body', () => {
+        render(<UnlinkAccountModal {...mock_props} />);
+
+        expect(
+            screen.getByText(
+                "To change your email address, you'll first need to unlink your email address from your Google account."
```

Comment:
maybe it would be more readable if we can move such long text to variable in order to make expectations fitting to one line